python/lsst/shapelet/__init__.py

```diff
@@ -89,3 +90,4 @@ def __iter__(self):
             if p == q:
                 yield (i, p, q, True)
                 i += 1
+
```

Comment:
I agree, but this was preexisting and there isn't an obvious place for it. How do we flag it?